homeassistant/components/sensor/teleinfo.py

```diff
@@ -0,0 +1,156 @@
+"""
+Support for reading Teleinfo data from a serial port.
+
+Teleinfo is a French specific protocol used in electricity smart meters.
+It provides real time information on power consumption, rates and current on
+a user accessible serial port.
+
+For more details about this platform, please refer to the documentation at
+https://www.enedis.fr/sites/default/files/Enedis-NOI-CPT_02E.pdf
```

Comment:
We should refer to the documentation on home-assistant.io.

Reply:
ok